test/lib/mocked-device-api.ts

```diff
@@ -1,17 +1,14 @@
 import * as _ from 'lodash';
```

Comment:
Do you know why we need this file? Have you maybe give it some though on how we can get rid from it?

Reply:
I think if we follow your suggestion of having each endpoint call `doXX` as with `doRestart` and `doPurge` for example, we can move towards removing mocked-device-api.ts. The current existence of this file seems be to because the Supervisor API interacts too closely with its underlying interfaces and interacts at multiple points in some routes, too, so shrinking these interfaces down to one function call will go a long way. The subject of another PR, perhaps?

Reply:
Moved with https://github.com/balena-os/balena-supervisor/commit/5714830fceef45714312d08aeb5229b8324eb2c7

Reply:
This will be done in the next API PR which will also move tests to unit / integration.